lib/datashake-ruby-sdk/review_index/v1/base_endpoint.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Datashake
+  module ReviewIndex
+    class V1
+      class BaseEndpoint
+        def self.allow_parameters(allowed_parameters)
+          allowed_parameters.each do |param_name|
```

Comment:
I wondering if `allowed_parameters` is bit vague name, maybe `request_parameters` would be more descriptive?

Reply:
Yeah, the last one sounds really better.